### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by davidk81**
- update readme for `--stats`

### CODE REVIEW

**Comment by davidk81** [comment]
should we mark the `--stats` feature as "beta" or "experimental" ?
just in case the numbers are not accurate

File: docs/stellar_satellite_open-stream.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ stellar satellite open-stream [satellite-id] [flags]
       --proxy string               Proxy protocol. One of: udp|tcp (default "udp")
       --send-host string           Deprecated: use udp-send-host instead.
       --send-port uint16           Deprecated: use udp-send-port instead.
+      --stats                      Output telemetry stats information and generates pass reports. (default false)
```
Comment (woobianca) [inline docs/stellar_satellite_open-stream.md]:
```suggestion
      --stats                      Output telemetry stats information and generate pass summaries. (default false)
```

File: cmd/flag/stats_flag.go [hunk 2]
```diff
@@ -23,7 +23,7 @@ type StatsFlag struct {
 
 // Add flags to the command.
 func (f *StatsFlag) AddFlags(cmd *cobra.Command) {
-	cmd.Flags().BoolVarP(&f.ShowStats, "stats", "", false, "Output telemetry stats information. (default false)")
+	cmd.Flags().BoolVarP(&f.ShowStats, "stats", "", false, "Output telemetry stats information and generates pass reports. (default false)")
```
Comment (woobianca) [inline cmd/flag/stats_flag.go]:
```suggestion
	cmd.Flags().BoolVarP(&f.ShowStats, "stats", "", false, "Output telemetry stats information and generate pass summaries. (default false)")
```

Comment (woobianca) [reply cmd/flag/stats_flag.go]: I feel like `report` sounds like it might contain an evaluation of the pass (quality, etc)

**Comment by danada** [comment]
Just double checking. Were these updates automatically generated? We have a tool to do this: https://github.com/infostellarinc/stellarcli/blob/master/util/docs/markdown/gen_markdown.go

Also, I think we should label the feature as alpha status since we may tweak the way we display stats. I'm afraid people may start consuming these stats using some tools and we'd break their tooling if we changed what we display.

**Comment by davidk81** [comment]
nay, was manual. 
noted

File: docs/stellar_satellite_open-stream.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ stellar satellite open-stream [satellite-id] [flags]
 ### Options
 
 ```
-      --accepted-framing strings   Framing type to receive. One of: BITSTREAM|AX25|IQ|IMAGE_PNG|IMAGE_JPEG|FREE_TEXT_UTF8|WATERFALL
+      --accepted-framing strings   Framing type to receive. One of: IQ|IMAGE_PNG|IMAGE_JPEG|FREE_TEXT_UTF8|WATERFALL|BITSTREAM|AX25
```
Comment (danada) [inline docs/stellar_satellite_open-stream.md:18]: For some reason the order of the values keep getting switched around 😆wonder what's causing it.

**Review by danada — APPROVED** [review]
Thanks for these improvements! Will look into releasing shortly.

**Pull request merged**